tests/scripts/ci.py

```diff
@@ -239,20 +231,20 @@ def docs(
     skip_build -- skip build and setup scripts
     interactive -- start a shell after running build / test scripts
     """
-    config = "./tests/scripts/task_config_build_gpu.sh"
     build_dir = get_build_dir("gpu")
     if cpu and full:
```

Comment:
should we change the cpu flag name here too? seems like it's more like tutorials or not

Reply:
not necessarily since tutorials don’t always require the GPU, though it’s probably usually used they way you mention, i’d prefer keeping it closer to what’s actually going on so it’s more clear how the tool works

Reply:
but doesn't cpu flag now imply ci_docs, not ci_cpu? just thinking it's named a bit weird

Reply:
resolved offline, removed the `--cpu` flag so the user doesnt need to directly state the image and its implied from whether the tutorials are built or not